### PR TITLE
CODEOWNERS, PR template

### PR DESCRIPTION
**Pull request by Amet13**
- PR template
- CODEOWNERS
- Cleanup

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* @Arriven
+# /docs/ @iamtodor @Amet13
```
Comment (Amet13) [inline .github/CODEOWNERS:2]: Commented due to lack of permission for review.
But we can keep comments for information who to ask

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Description
```
Comment (Amet13) [inline .github/pull_request_template.md:1]: Probably, we can get more useful info in PRs and reduce time to review and ask questions.
If it works well, can do similar for Issues as well

File: .github/workflows/test-installer.yaml [hunk 3]
```diff
@@ -1,12 +1,12 @@
+name: Test install.sh
+
 on:
   push:
     branches:
       - "main"
-  pull_request:
```
Comment (Amet13) [inline .github/workflows/test-installer.yaml:5]: Eventually run installation test only on merge to main

File: FAQ.md [hunk 4]
```diff
@@ -36,4 +36,4 @@ The main reason it works that way is because there are two main types of ddos:
 TL;DR: [yes](https://github.com/Arriven/db1000n/issues/153)
 
 Cloud providers could charge a huge amount of money not only for compute resources but for traffic as well.
-If you run an app in the cloud please control your billing
+If you run an app in the cloud please control your billing (if you use Docker, ensure that use advanced image: `ghcr.io/arriven/db1000n-advanced`)
```
Comment (Amet13) [inline FAQ.md:39]: Mentioned before

File: Makefile [hunk 5]
```diff
@@ -1,10 +1,2 @@
 build:
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o main ./main.go
-docker_image:
```
Comment (Amet13) [inline Makefile:3]: I believe it's useless, not sure about build target, but if it's also useless, we can remove the whole Makefile

File: docs/docker-vpn.md [hunk 6]
```diff
@@ -1,108 +1,9 @@
 # Setting up VPN for Docker users
 
-In case of using a dedicated VPS that has banned public IP, a container with
-OpenVPN client can be deployed inside the same network as db1000n is in.
-One of the easy ways to set it up is through the docker compose:
+In case of using a dedicated VPS that has banned public IP, a container with OpenVPN client can be deployed inside the same network as db1000n is in.
+One of the easy ways to set it up is through the docker-compose.
 
-`docker-compose.yml`:
```
Comment (Amet13) [inline docs/docker-vpn.md:7]: Copy-paste from docker-compose.yml, just left a reference to a file

File: .github/CODEOWNERS (hunk 1 above)
Comment (iamtodor) [reply .github/CODEOWNERS:2]: @Amet13 perhaps @Arriven could provide specific access according to the `least privilege principle`?

**Comment by iamtodor** [comment]
@Arriven may we ask with @Amet13 to have `Triage` role in the repo? According to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository doc this role allows to manage issues and pull requests

**Comment by iamtodor** [comment]
@Arriven relates to https://github.com/Arriven/db1000n/pull/187#pullrequestreview-901101286 as well

File: .github/CODEOWNERS (hunk 1 above)
Comment (Arriven) [reply .github/CODEOWNERS:2]: I'll take a look, can't remember if regular user repos allow access differentiation

**Review by Arriven — APPROVED** [review]

**Comment by iamtodor** [comment]
@Arriven can we discuss providing certain restricted access to the repo? How can I reach you out?
email or telegram?
